pkg/apis/management.cattle.io/v3/pod_security_types.go

```diff
@@ -0,0 +1,92 @@
+package v3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const PodSecurityAdmissionConfigurationTemplateKind = "PodSecurityAdmissionConfigurationTemplate"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type PodSecurityAdmissionConfigurationTemplate struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Configuration PodSecurityAdmissionConfigurationTemplateSpec `json:"configuration"`
```

Comment:
I know it's not part of the requirements from the linked issue, but would adding `description` (we have it on `PodSecurityPolicyTemplate`) be beneficial here to explain the purpose of each PSAConfigTemplate?

Reply:
After discussing we agreed to add a new field `description` in a similar fashion to how PSP's are implemented (I initially thought this request was to add a comment description to the field).